### PR TITLE
replacing swapfile role

### PR DESCRIPTION
**Pull request by speedlight**
ISSUE: #124 

Replacing swapfile role with geerlingguy.swap;
Adding condition to enable/disable role instalation for molecule.

### CODE REVIEW

File: molecule/default/requirements.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 ---
-- src: kamaln7.swapfile
-  version: "0.4"
+- src: geerlingguy.swap
```
Comment (sestrella) [inline molecule/default/requirements.yml]: @speedlight I think we should target a specific version in order to have more control over the dependencies and gradually upgrade them.

File: molecule/default/playbook.yml [hunk 2]
```diff
@@ -11,3 +11,4 @@
     - role: sb-debian-base
       vars:
         prebootstrap: true
+        swapfile: false
```
Comment (sestrella) [inline molecule/default/playbook.yml]: @speedlight what is the purpose behind this flag?

File: molecule/default/requirements.yml (hunk 1 above)
Comment (sestrella) [reply molecule/default/requirements.yml]: @speedlight additionally, I think we need to upgrade the role version

Comment (speedlight) [reply molecule/default/requirements.yml]: Thanks @sestrella, is fixed now.

Comment (speedlight) [reply molecule/default/requirements.yml]: But I need to update the version.. I understand taht with this change is only a minor version update? like 1.6.1, I'm right? @sestrella

File: molecule/default/playbook.yml (hunk 2 above)
Comment (speedlight) [reply molecule/default/playbook.yml]: Sorry non related to this, was testing enabling bootstrap and swapfile :)

File: molecule/default/requirements.yml (hunk 1 above)
Comment (speedlight) [reply molecule/default/requirements.yml]: I push a v.1.6.1 version tag to reflect the change in swap role dependecy.

File: .python-version [hunk 3]
```diff
@@ -0,0 +1 @@
+3.7.0
```
Comment (sestrella) [inline .python-version]: @speedlight there is already a [.tool-versions](https://github.com/stackbuilders/sb-debian-base/blob/master/.tool-versions) file in place, which flags the Python version used by this project. `asdf` has a feature called `legacy_version_file` which allows the tool to ready Python version from `.python-version`, if you consider worth it, we could turn it on for this project and get rid of `.tool-versions`, but maybe we could tackle that in a separate PR. What do you think about getting rid of this file for now?

https://asdf-vm.com/#/core-configuration?id=homeasdfrc

File: meta/main.yml [hunk 4]
```diff
@@ -24,11 +24,11 @@ galaxy_info:
...
 dependencies:
-  - src: kamaln7.swapfile
-    version: "0.4"
-    swapfile_size: "{{ sb_debian_base_swap_file_size }}"
+  - src: geerlingguy.swap
+    swap_file_size_mb: "{{ sb_debian_base_swap_file_size }}"
     become: yes
-    when: bootstrap is defined
+    when: |
+      bootstrap is defined
```
Comment (sestrella) [inline meta/main.yml]: @speedlight is this line break necessary?

File: meta/main.yml [hunk 5]
```diff
@@ -24,11 +24,11 @@ galaxy_info:
     - ubuntu
 
 dependencies:
-  - src: kamaln7.swapfile
-    version: "0.4"
-    swapfile_size: "{{ sb_debian_base_swap_file_size }}"
+  - src: geerlingguy.swap
+    swap_file_size_mb: "{{ sb_debian_base_swap_file_size }}"
```
Comment (sestrella) [inline meta/main.yml]: @speedlight should we point to a specific version here?

File: .python-version (hunk 3 above)
Comment (speedlight) [reply .python-version]: Definetly! Sorry the for upload, this is not suposed to happend :(

File: meta/main.yml (hunk 4 above)
Comment (speedlight) [reply meta/main.yml]: No is not necessary, is my habit to put conditionals in separates lines :) I fixed this, thanks for the observation.

File: meta/main.yml (hunk 5 above)
Comment (speedlight) [reply meta/main.yml]: Yes @sestrella thanks for pointed out.

File: CHANGELOG.md [hunk 6]
```diff
@@ -7,6 +7,7 @@
 
 ## 1.6.1
 * Add webhooks into travis.yml to notifiy galaxy about new releases.
+* Replace kamaln7.swapfile with geerlingguy.swap requeriment role.
```
Comment (sestrella) [inline CHANGELOG.md]: @speedlight I think we need to update the CHANGELOG since we are currently at version v1.6.2

**Pull request merged**